### ISSUE TITLE
Accuracy instead Perplexity?

### ISSUE THREAD

**Issue by colino** [Microsoft/CNTK#648]
Hi,
how can i show accuracy instead of perplexity in output?

**Comment by frankseide** [comment]
Normally, training jobs are configured to show two numbers: the training objective and an error count. For example:

    ce = CrossEntropyWithSoftmax(labels, ol)
    errs = ErrorPrediction(labels, ol)

    criterionNodes = (ce)
    evalNodes = (errs)

`criterionNodes` should list one node, which determines the objective that is optimized for. The `evalNodes` list nodes that merely get evaluated during training. They will only affect the training through cross-validation based methods such as learning-rate adjustment.

What exactly is your job? E.g. if it is language modeling, the notion of an error rate may not be meaningful, since one would normally predict entire sequences, rather than a single word based on a ground-truth history (which is what `ErrorPrediction()` would give you).
